src/MoBi.Engine/Sbml/FunctionDefinitionImporter.cs

```diff
@@ -32,5 +32,10 @@ protected override void Import(Model model)
       public override void AddToProject()
       {
       }
+
+      public void Initialize()
+      {
+         _functionDefinitions.Clear();
```

Comment:
is this class FunctionDefinitionImporter singleton? Why isn't it done in constructor?
We have a IStartable interface also... maybe that should be enough?

Reply:
It is a singleton yes. Then if you load a file and load a second file, it will remember all the units and functions defined so you need to clearly trigger "this is a new file" and reset to a clean state. This is why for example tests were failing.